packages/components/scripts/wrapper-generator/UXPinReactWrapperGenerator.ts

```diff
@@ -440,6 +442,39 @@ export class UXPinReactWrapperGenerator extends ReactWrapperGenerator {
     return [...componentPresetFiles, configFile];
   }
 
+  // Component declaration can be preceded by JSDoc comments
+  // to customize the behavior in UXPin Editor or Preview (E.g.: render in a React Portal)
+  // https://www.uxpin.com/docs/merge/adjusting-components/
+  private insertComponentAnnotation(cleanedComponent: string, component: TagName): string {
+    const comments = this.getAllComponentComments(component);
+    if (!comments.length) return cleanedComponent;
```

Comment:
Let's please avoid same line return and add an `else` case for better readability.

Reply:
Done here https://github.com/porsche-design-system/porsche-design-system/pull/2208/commits/959c331a17d0b714cfaf3aec8ccd5b04442b42b0

I'm always happy to clean stuff 😄